lib/api.js

```diff
@@ -171,14 +188,13 @@ module.exports.multipartUpload = async function (options, uploadId, fileName, fi
         } catch (err) {
           // S3 will respond with a 400 for a socket timeout which could occur from a slow stream read.
           // Retry these errors up to 10 times to get the part uploaded.
-          if (err.response.status === 400) {
```

Comment:
I found that a part would be attempted 10 times, ++attempts, then we'd exit the do while loop without an error.  I've modified this to throw once we reach max attempts (still at 10).  Let me know if I misunderstood the intention here and we want to avoid throwing, but it felt like a silent failure when I saw it happen from the CLI.